### PR TITLE
Finsh register page and part of club detail page

### PR DESCRIPTION
**Pull request by wangwenyue**
Finsh register page.

Add link button for page switch.

Finsh club member page UI.

Add join club Modal in personal page.

### CODE REVIEW

File: src/components/Header/index.js [hunk 1]
```diff
@@ -7,13 +8,13 @@ import {
 class Header extends React.Component {
 
     render() {
-        let titleName = this.props.titleName
-        return(
+        const titleName = this.props.titleName
```
Comment (Keith-CY) [inline src/components/Header/index.js]: use `deconstruct` as `const { titleName } = this.props`

File: src/containers/Detail/ClubMember/index.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+class ClubMember extends React.Component {
+    //TODO add club leader banner
+    render() {
+        const {
+            memberDataList,
+        } = this.props.clubmemberStore
+
+        const memberList = memberDataList.map((data, index) => {
+            return (
+                <Item
```
Comment (Keith-CY) [inline src/containers/Detail/ClubMember/index.js:21]: component items in an array should be labelled with `key`

File: src/containers/Detail/detail.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
+.club-detail-pannel {
+    height: 170px;
+    /*border: 1px red solid;*/
+    background-color: #fff;
+}
+
+.club-detail-name {
+    width: 100vw;
+    text-align: center;
+    font-family: PingFangSC, Arial, sans-serif, serif;
```
Comment (Keith-CY) [inline src/containers/Detail/detail.css]: repeated properties could be replaced with css variables.

File: src/containers/Detail/index.js [hunk 4]
```diff
@@ -1,12 +1,51 @@
-import React from 'react';
-import {observer, inject} from 'mobx-react';
+import React from 'react'
+import {observer, inject} from 'mobx-react'
+import {Header} from '../../components'
+import {
```
Comment (Keith-CY) [inline src/containers/Detail/index.js:3]: imports from node_modules should be placed together

File: src/containers/Detail/detail.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
+.club-detail-pannel {
```
Comment (Keith-CY) [inline src/containers/Detail/detail.css]: `pannel` or `panel`, don't use both of them

File: src/containers/Detail/index.js [hunk 6]
```diff
@@ -1,12 +1,51 @@
...
+        } = this.props.detailStore
+
         return (
-            <div>Club Details page</div>
+            <div className='detail-container'>
+                <Header
+                    titleName='社团详情'
+                    backRoute='./personal'
+                />
+                <div className='club-detail-pannel'>
```
Comment (Keith-CY) [inline src/containers/Detail/index.js]: `pannel` or `panel`, don't use both of them

File: src/containers/Personal/ClubListItem/index.js [hunk 7]
```diff
@@ -3,20 +3,20 @@ import {observer, inject} from 'mobx-react'
...
-            arrowRight,
+                handleClubDetail,
+                clubDataList,
+                arrowRight,
         } = this.props.clublistStore
 
-        const clubList = dataList.map((data, index) => {
+        const clubList = clubDataList.map((data, index) => {
             return (
                 <Flex onClick={handleClubDetail} className='club-list-item'>
```
Comment (Keith-CY) [inline src/containers/Personal/ClubListItem/index.js]: add `key` to items in an array

File: src/containers/Personal/PersonalPanel/index.js [hunk 8]
```diff
@@ -5,6 +5,7 @@ import {
     Flex,
     List,
 } from 'antd-mobile'
+import {Link} from "react-router-dom"
```
Comment (Keith-CY) [inline src/containers/Personal/PersonalPanel/index.js]: space around `Link`

File: src/stores/personalStore/index.js [hunk 9]
```diff
@@ -1,29 +1,33 @@
 import {observable, action} from 'mobx'
 import React from "react";
```
Comment (Keith-CY) [inline src/stores/personalStore/index.js]: semicolon could be removed

File: src/stores/personalStore/index.js [hunk 10]
```diff
@@ -1,29 +1,33 @@
...
+import {
+    Modal,
+} from 'antd-mobile'
 
 const log = console.log.bind(console, '### personalStore ')
 
 const thumbPic = 'avatar.png'
 const joinIcon = <img src="per_join.png" alt=""/>
 const createIcon = <img src="per_create.png" alt=""/>
+const prompt = Modal.prompt
```
Comment (Keith-CY) [inline src/stores/personalStore/index.js]: use `deconstruct` as `const { prompt } = Modal`

**Review by Keith-CY — CHANGES_REQUESTED** [review]

**Comment by wangwenyue** [comment]
@Keith-CY All requests resolved.

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**